priam/src/test/java/com/netflix/priam/FakeConfiguration.java

```diff
@@ -914,4 +914,12 @@ public String getPostRestoreHookDoneFileName() {
     public int getPostRestoreHookTimeOutInDays() {
         return 2;
     }
+
+    @Override
+    public int getPostRestoreHookHeartBeatTimeoutInMs() {
```

Comment:
No need to provide this if you provide default implementation in IConfiguration.